content/general/limits.textile

```diff
@@ -78,8 +78,8 @@ The quotas for an account can be considered a soft limit, with no restrictions c
 
 |_/2=. Limit |_\2=. Free |_\2=. PAYG |_\2=. Committed Use |
 |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |
-| "Peak connections":#peak-connections | 200 | 240 | 10,000 | 25,000 |\2. Custom >= PAYG |
-| "Peak channels":#peak-channels | 200 | 240 | 10,000 | 25,000 |\2. Custom >= PAYG |
+| "Peak connections":#peak-connections | 200 | 240 | 10,000 | 250,000 |\2. Custom >= PAYG |
```

Comment:
I think the soft limit for peak connections should also be 100,000 here.

Reply:
I believe I updated this! Whew